docs/pages/reference/authentication.mdx

```diff
@@ -121,6 +121,50 @@ $ tctl create -f cap.yaml
 </TabItem>
 </Tabs>
 
+### Local user locks
+
+A lock is placed on a local user if, within a 30 minute window, a local user has:
+
+- 5 failed login attempts or
```

Comment:
It's really easy for this to become out of sync with the code. I wonder if we need to mention the specifics or if we can just say that teleport will lock accounts after too many failed logins or password resets.

Reply:
Please see update